### PR TITLE
sane-backends: 1.0.32 -> 1.1.1

### PR DESCRIPTION
**Pull request by symphorien**
and add myself as maintainer

changelog: https://gitlab.com/sane-project/backends/-/releases/1.1.1, nothing of note

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/graphics/sane/backends/default.nix [hunk 1]
```diff
@@ -16,17 +16,19 @@
...
     # raw checkouts of the repo do not work because, the configure script is
     # only functional in manually uploaded release tarballs.
     # https://gitlab.com/sane-project/backends/-/issues/440
     # unfortunately this make the url unpredictable on update, to find the link
     # go to https://gitlab.com/sane-project/backends/-/releases and choose
-    # the link with other in the URL.
-    url = "https://gitlab.com/sane-project/backends/uploads/104f09c07d35519cc8e72e604f11643f/sane-backends-1.0.32.tar.gz";
-    sha256 = "055iicihxa6b28iv5fnz13n67frdr5nrydq2c846f9x7q0vw4a1s";
+    # the link under the heading "Other".
+    src = fetchurl {
```
Comment (QuantMint) [inline pkgs/applications/graphics/sane/backends/default.nix:28]: Seems like this line got duplicated

Comment (symphorien) [reply pkgs/applications/graphics/sane/backends/default.nix]: ahah I had fixed it locally but forgot to force push... thanks!

**Comment by ajs124** [comment]
duplicate of #195568, but I'm inclined to go with this PR instead of the old one, because you're adding yourself as a maintainer here.

cc @heinrich5991

**Comment by heinrich5991** [comment]
Fine by me.

**Comment by symphorien** [comment]
sorry to steal your first contribution @heinrich5991, I hope you will have new occasions to contribute :)

**Pull request merged**